packages/node-client/lib/index.ts

```diff
@@ -680,23 +702,23 @@ export class Nango {
     /**
      * Override a sync’s default frequency for a specific connection, or revert to the default frequency
      * @param providerConfigKey - The key identifying the provider configuration on Nango
-     * @param sync - The name of the sync to update
+     * @param sync - The name of the sync to update (or an object with name and variant properties)
      * @param connectionId - The ID of the connection for which to update the sync frequency
      * @param frequency - The new frequency to set for the sync, or null to revert to the default frequency
      * @returns A promise that resolves with the response data after updating the sync frequency
      */
     public async updateSyncConnectionFrequency(
         providerConfigKey: string,
-        sync: string,
+        sync: string | { name: string; variant: string },
```

Comment:
maybe a type for the param could have created

Reply:
I couldn't find a good name so I went for the solution of not naming it :) do you have any idea?

Reply:
not really 😓